orb-ui/src/sound/capture.rs

```diff
@@ -0,0 +1,146 @@
+use crate::sound;
+
+/// Loop sounds during biometric capture
+/// divided into 3 loops with 3 parts each
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum CaptureLoopSound {
+    Loop01(LoopSoundPart),
+    Loop02(LoopSoundPart),
+    Loop03(LoopSoundPart),
+}
+
+/// Loop sound sub-parts
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum LoopSoundPart {
+    PartA,
+    PartB,
+    PartC,
+}
+
+impl Default for CaptureLoopSound {
+    fn default() -> Self {
+        CaptureLoopSound::Loop01(LoopSoundPart::PartA)
+    }
+}
+
+impl CaptureLoopSound {
+    /// Come back to part A when out of range detected
+    pub fn out_of_range(&mut self) {
```

Comment:
the name of this function makes it sound like this detects when it is out of range, rather than "do thing when we are out of range".

Lets think of a more descriptive name, Maybe, restart_loop?